service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -131,6 +142,7 @@ void aboutToStartShouldSetDateOfIssueAsTodayByDefault() {
         @Test
         void aboutToSubmitShouldAddC21OrderToCaseDataAndRemoveTemporaryCaseDataOrderFields() {
             final CaseDetails caseDetails = buildCaseDetails(commonCaseDetailsComponents(BLANK_ORDER, null)
+                .caseLocalAuthority(LOCAL_AUTHORITY_CODE)
```

Comment:
why did this need to be added here? Maybe I will come to it in later code 🤷‍♂

Reply:
I added this because loads of the tests in this class where failing on null pointer, looking for a local authority so thought i'd fix?

Reply:
not tests i'd added but previous ones

Reply:
why do the changes you've made require this?

Reply:
discussed